src/modules/ordersTable/containers/OrdersTableWidget/index.tsx

```diff
@@ -46,12 +46,12 @@ const ContentWrapper = styled.div`
   width: 100%;
 `
 
-export function OrdersTableWidget() {
+export function OrdersTableWidget({ customOrders }: { customOrders?: ParsedOrder[] }) {
   const { chainId, account } = useWalletInfo()
   const location = useLocation()
   const navigate = useNavigate()
   const allOrders = useOrders({ chainId })
```

Comment:
shouldn't `customOrders` be part of `allOrders`?  (instead of being passed by param)